### DISCUSSION TITLE
JsonObject unable to display symbol such as + < >

### DISCUSSION THREAD

**Discussion by tastynoob** [dotnet/csharplang#8196] [Q&A]
Very strange
the code:
```
JsonObject obj = new(){
  {"test" , "+"}
};
Console.WriteLine(obj.ToJsonString());
```
it's output
```
{"test":"\u002B"}
```

i found the symbols '+' '<' '>' all unable to display

if
```
JsonObject obj = new(){
  {"test" , "+="}
};
Console.WriteLine(obj.ToJsonString());
```
it's output
```
{"test":"\u002B="}
```

**Comment by CyrusNajmabadi** [comment]
This is not a language question. You might want to ask the authors of JsonObject.

**Reply by tastynoob** [reply]
wait, I use "using System.Text.Json.Nodes", Isn't this build-in packet?

**Reply by jnm2** [reply]
That would be https://github.com/dotnet/runtime.

**Reply by tastynoob** [reply]
Ok, thanks
